build/tools/esp32/esptool_py/Makefile

```diff
@@ -46,13 +46,13 @@ APP_BIN:=$(APP_ELF:.elf=.bin)
 # two commands that can be used from other components
 # to invoke esptool.py (with or without serial port args)
 #
-ESPTOOLPY_SRC := $(CURRENTDIR)/esptool_py/esptool.py
+ESPTOOLPY_SRC := $(CURRENTDIR)/esptool_py/esptool/esptool.py
 ESPTOOLPY := $(PYTHON) $(ESPTOOLPY_SRC) --chip esp32
 ESPTOOLPY_SERIAL := $(ESPTOOLPY) --port $(ESPPORT) --baud $(ESPBAUD) --before $(CONFIG_ESPTOOLPY_BEFORE) --after $(CONFIG_ESPTOOLPY_AFTER)
 
 # Supporting esptool command line tools
-ESPEFUSEPY := $(PYTHON) $(CURRENTDIR)/esptool/espefuse.py
-ESPSECUREPY := $(PYTHON) $(CURRENTDIR)/esptool/espsecure.py
+ESPEFUSEPY := $(PYTHON) $(CURRENTDIR)/esptool_py/esptool/espefuse.py
```

Comment:
Useless definition in Makefiles.

Reply:
> @zhouxinhe,
> Are supporting secure boot now? Is espsecure.py a immediate need?
> Is is possible to get the download tools (under another license) from expressif?

- secure boot is not supported now, I deleted definitions related to espsecure.py&espefuse.py.
- there's download tools espressif provided, @xiarihf is verifying it and will share with you later.

Reply:
Fixed